modules/nodes-descheduler-installing.adoc

```diff
@@ -46,3 +65,4 @@ Do not enable both `TopologyAndDuplicates` and `SoftTopologyAndDuplicates`. Enab
 .. Click *Create*.
 
 You can also configure the profiles and settings for the descheduler later using the OpenShift CLI (`oc`). If you did not adjust the profiles when creating the descheduler instance from the web console, the `AffinityAndTaints` profile is enabled by default.
+endif::nodes[]
```

Comment:
Unset the attributes at the end of the module:

ifeval::["{context}" == "nodes-descheduler"]
:!nodes:
endif::[]

ifeval::["{context}" == "virt-enabling-descheduler-evictions"]
:!virt:
endif::[]